Homework3/smartQsort.c

```diff
@@ -0,0 +1,134 @@
+#include <stdio.h>
+#include <time.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+void fillRandomArray(unsigned int arraySize, int array[]) {
+    if (arraySize < 1) {
+        return;
+    }
+    
+    srand((unsigned)time(0));
```

Comment:
```suggestion
    srand((unsigned)time(NULL));
```
time указатель принимает